src/components/Input/Button.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import type { PlacesType } from 'react-tooltip';
```

Comment:
Can this be moved to below line 7 (with a one line gap after 7)?
If linter complains, move it back 😔

Reply:
Unfortunately: `react-tooltip type import should occur before import of @mdi/js  import/order`